postprocessing/land_utils/combine-ncc.F90

```diff
@@ -1,7 +1,26 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the GFDL FRE NetCDF tools package (FRE-NCTools).
+!*
+!* FRE-NCTools is free software: you can redistribute it and/or modify it under
+!* the terms of the GNU Lesser General Public License as published by
+!* the Free Software Foundation, either version 3 of the License, or (at
+!* your option) any later version.
+!*
+!* FRE-NCTools is distributed in the hope that it will be useful, but WITHOUT
+!* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+!* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+!* for more details.
+!*
+!* You should have received a copy of the GNU Lesser General Public
+!* License along with FRE-NCTools (LICENSE.md) .  If not, see
```

Comment:
I'd prefer not list the file that contains the license.  Although I doubt it will, it could change in the future.

Reply:
I  have the same preference. Will remove the "(LICENCSE.md") text if there are no objections.

Reply:
I was the one who had suggested it (based on some example licenses on the Internet) but I agree, no need to box ourselves into a filename.

Reply:
I have removed the  "(LICENCSE.md") text.